src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="xunit.extensibility.core" Version="$(XunitVersion)" />
```

Comment:
So, yes it can... But `XunitVersion` is set to `2.2.0`, but `xunit.abstractions` is only version `2.0.1`. I wasn't sure which was less bad: 1) Adding an `XunitAbstractionsVersion` variable, 2) Setting `xunit.abstractions` to `2.0.1` without a variable or 3) Using `xunit.extensibility.core`. Very open to suggestions :)